modules/account-assignments/variables.tf

```
@@ -7,3 +7,9 @@ variable "account_assignments" {
     principal_type      = string
   }))
 }
+
+variable "wait_group_creation" {
```

why this var name? it does not tell what the var does.
Please change it to `depends_on`.
```suggestion
variable "depends_on" {
```

Also, why do we need a list here if the values are joined together into a string in 

```
triggers = {
    dependency_id = join(",", var.wait_group_creation)
  }
```

---

Hi @aknysh

About the name change, won't it be confusing with the "builtin" depends on? The name came from the examples links in the description. 
No strong feelings about it, whatever is decided is fine with me.

Regarding the list, please see this thread:
https://github.com/cloudposse/terraform-aws-sso/pull/33#discussion_r1052657605

---

@simonweil 

1. We used that pattern before and named the variable `depends_on`, so it should be ok to rename it
2. Why do we need to make the variable `wait_group_creation` a list? It gets converted to a string in `join(",", var.wait_group_creation)` anyway, so prob it could be just a string. Or maybe you have some examples on how you use it and you require a list of strings as dependencies? (if yes, please share it)

---

please also run 

```
make init
make github/init
make readme
```

---

1. It doesn't work, I get the following error
```
✗ terraform validate
╷
│ Error: Invalid variable name
│
│   on variables.tf line 11, in variable "depends_on":
│   11: variable "depends_on" {
│
│ The variable name "depends_on" is reserved due to its special meaning inside module blocks.
╵
```
Tell me if I should change the name to something else

2. see the reasoning here: https://github.com/cloudposse/terraform-aws-sso/pull/33#discussion_r1053598577, just tell me what to do, I'm fine with it this way or that way

3. `make` ran and pushed